app/src/main/java/com/easyrouteapp/service/RoutesService.java

```diff
@@ -0,0 +1,25 @@
+package com.easyrouteapp.service;
+
+import com.easyrouteapp.dto.FilterDto;
+import com.easyrouteapp.dto.ReturnDataRouteDto;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.Header;
+import retrofit2.http.Headers;
+import retrofit2.http.POST;
+
+/**
+ * Created by fernando on 11/08/2016.
+ */
+public interface RoutesService {
```

Comment:
On the last version I changed the rest requests to use retrofit library. It was really better then using the simple http Java connection and made my code more easily to understand and maintain.